homeassistant/components/tessie/config_flow.py

```diff
@@ -0,0 +1,57 @@
+"""Config Flow for Tessie integration."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+from http import HTTPStatus
+from typing import Any
+
+from aiohttp import ClientConnectionError, ClientResponseError
+from tessie_api import get_state_of_all_vehicles
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ACCESS_TOKEN
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+TESSIE_SCHEMA = vol.Schema({vol.Required(CONF_ACCESS_TOKEN): str})
+
+
+class TessieConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config Tessie API connection."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: Mapping[str, Any] | None = None
+    ) -> FlowResult:
+        """Get configuration from the user."""
+        errors: dict[str, str] = {}
+        if user_input and CONF_ACCESS_TOKEN in user_input:
+            try:
+                await get_state_of_all_vehicles(
+                    session=async_get_clientsession(self.hass),
+                    api_key=user_input[CONF_ACCESS_TOKEN],
+                    only_active=True,
+                )
+            except ClientResponseError as e:
+                if e.status == HTTPStatus.UNAUTHORIZED:
+                    errors["base"] = "invalid_access_token"
```

Comment:
```suggestion
                    errors[CONF_ACCESS_TOKEN] = "invalid_access_token"
```

Reply:
I didnt know thats how that works. Cool.